lemur/certificates/service.py

```diff
@@ -344,8 +344,9 @@ def create_csr(**csr_config):
     private_key = generate_private_key(csr_config.get('key_type'))
 
     builder = x509.CertificateSigningRequestBuilder()
-    name_list = [x509.NameAttribute(x509.OID_COMMON_NAME, csr_config['common_name']),
-                 x509.NameAttribute(x509.OID_EMAIL_ADDRESS, csr_config['owner'])]
+    name_list = [x509.NameAttribute(x509.OID_COMMON_NAME, csr_config['common_name'])]
+    if current_app.config.get('LEMUR_OWNER_EMAIL_IN_SUBJECT', True):
```

Comment:
Do you think this should even default to `True`?

Reply:
Have you encountered CAs that don't allow this? Or is this purely from a privacy consideration?

Reply:
It's for privacy, yes.

Reply:
I would then leave the default as `True` and let them opt out of it. So as not to break the existing functionality.

Reply:
Sure, works for me